velox/type/Filter.h

```diff
@@ -28,6 +28,7 @@
 #include "velox/common/base/Exceptions.h"
 #include "velox/common/base/SimdUtil.h"
 #include "velox/type/StringView.h"
+#include "velox/type/UnscaledLongDecimal.h"
```

Comment:
Is this header required?

Reply:
The header is added to use type name ```int128_t```. Please let me know if ```__int128_t``` should be used instead.